src/parser/TTML.cpp

```diff
@@ -144,10 +144,15 @@ start(void *data, const char *el, const char **attr)
 static void XMLCALL
 text(void *data, const char *s, int len)
 {
-  TTML2SRT *ttml(reinterpret_cast<TTML2SRT*>(data));
-  
-  if (ttml->m_node & TTML2SRT::NODE_P)
-    ttml->m_strXMLText += std::string(s, len);
```

Comment:
Please reduce code complexity for this simple check:

If (len > 1 || *s != '\n') would be sufficiant here.
There is no need to copy the const char* into a std::string